src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -26,6 +26,7 @@
   --pf-c-context-selector__toggle-text--MaxWidth: calc(var(--pf-c-context-selector--Width) - var(--pf-c-context-selector__toggle-icon--MarginLeft) - 30px); // hard coded last variable
```

Comment:
Apologies, this isn't new to this PR, but this comment needs to be more descriptive. Why is it hardcoded to 30px?

Reply:
Yes updated this thanks for catching that!!